app/src/js/actions/index.js

```diff
@@ -944,3 +944,4 @@ export const filterReconciliationReport = (param) => ({ type: types.FILTER_RECON
 export const clearReconciliationReportFilter = (paramKey) => ({ type: types.CLEAR_RECONCILIATION_FILTER, paramKey });
 
 export const toggleSidebar = () => ({ type: types.TOGGLE_SIDEBAR });
+export const sortPersist = () => ({ type: types.SORTS });
```

Comment:
```suggestion
export const sortPersist = (sortBy) => ({ type: types.SORTS, sortBy });
```

We want to pass information to the action so that the reducer can do something with it